woocommerce-google-analytics-integration.php

```diff
@@ -137,7 +138,7 @@ public function add_integration( $integrations ) {
 		 */
 		public function maybe_show_ga_pro_notices() {
 			// Notice was already shown
-			if ( get_option( 'woocommerce_google_analytics_pro_notice_shown', false ) ) {
+			if ( ! class_exists( 'WooCommerce' ) || get_option( 'woocommerce_google_analytics_pro_notice_shown', false ) ) {
```

Comment:
In the `get_option`, the `$default` equals `false` by default so you can omit it.